specs/remove-hypervisor-dependency.md

```diff
@@ -0,0 +1,45 @@
+# Remove hypervisor dependencies
+
+## Scope
+We want to make Open vStorage a real storage product which no longer depends on a hypervisor. In previous versions we always assumed that the Volume Driver ran together with a hypervisor (VMware/KVM). With the Edge this no longer the case as the edge client runs on the compute nodes (hypervisor) and the volume driver runs separately on a different node.
+With th FR we want to break this hypervisor dependency. This means you can now install Open vStorage and create a vdisk and add the hypervisor later.
```

Comment:
"... and add the hypervisor later". We had decided that there won't be any hypervisors left wrt our product so you can't "add" it later. Unless you mean that you can use a hypervisor of your liking, but not integrated with OVS whatsoever.